src/components/Home.js

```diff
@@ -26,25 +25,25 @@ const Home = (props) => {
       .get()
       .then((querySnapshot) => {
         if (querySnapshot.empty) {
-          alert("uh oh! list does not exist");
+          alert('uh oh! list does not exist');
         } else {
-          localStorage.setItem("token", token);
+          localStorage.setItem('token', token);
           props.setAuth(token);
-          history.push("/ListView");
+          history.push('/ListView');
         }
       })
       .catch((error) => {
         alert(error);
       });
 
-    setToken("");
+    setToken('');
   };
 
   return (
     <div className="home">
-      <h1>Welcome to Smart Shopping List!</h1>
+      <h1>Welcome to Smart-Shopper! Your shopping list is empty.</h1>
       <button className="new-list-btn" onClick={generateToken}>
-        Create a new list
+        Add item
```

Comment:
I think we should change this text back to "Create a new list" since that's what this is actually doing.